slog_test.go

```diff
@@ -63,8 +63,8 @@ func TestSlogHandler(t *testing.T) {
 		}); ok {
 			for _, err := range err.Unwrap() {
 				if !containsOne(err.Error(),
-					"a Handler should ignore a zero Record.Time",             // zapr always writes a time field.
-					"a Handler should not output groups for an empty Record", // Relies on WithGroup and that always opens a group. Text may change, see https://go.dev/cl/516155
+					"a Handler should ignore a zero Record.Time",                    // zapr always writes a time field.
+					"a Handler should not output groups if there are no attributes", // Relies on WithGroup and that always opens a group.
```

Comment:
We might have to keep the old string to avoid errors when using an old Go version.

That aside, https://github.com/golang/go/issues/61758 probably got merged long enough ago that we can skip unsupported sub-tests as outlined in https://github.com/golang/go/issues/65400#issuecomment-1955414622.

Can you give that a try instead?